app/src/androidMain/kotlin/features/sensor/SensorViewModel.kt

```diff
@@ -67,8 +67,8 @@ val ViewState.label: CharSequence
     get() = when (this) {
         ViewState.Connecting -> "Connecting"
```

Comment:
Should we show the connecting sub-states?

Reply:
Good thinking. If I remember to later, I'll add those.